app/components/Emoji.tsx

```diff
@@ -0,0 +1,39 @@
+import data from "@emoji-mart/data";
```

Comment:
We're going to have to see if this can replace the `gemoji` library usage too soon, these emoji libs are huge and now we have two + `emoji-regex` as well.

Reply:
PR: https://github.com/outline/outline/pull/4390